### ISSUE TITLE
Add `disableButtonStyles` option?

### ISSUE THREAD

**Issue by amillward** [Mastermindzh/react-cookie-consent#35]
I have what I believe to be a common use case; I want to use the default styles but just style the button (in my case with a bootstrap `btn btn-primary` class names). But if I provide `buttonClasses="btn btn-primary"`, it is overridden by the inline styles. If I add `disableStyles={true}`, I lose all the other styles.

Thoughts on adding a `disableButtonStyles` option to remove only the button styles?

**Comment by Mastermindzh** [comment]
Hey Abby,

Let me get this straight, you want to disable the styles for the button with a property while keeping the other styles intact? 
That seems like a solid plan! Buttons are often something I want to change on a theme basis as well. 

I can implement that tomorrow-ish, thanks for the feedback!

In the meantime you can work around it like so:

- Disable styles with `disableStyles={true}`
- In the style property pass:
```js
      style: {
        alignItems: "baseline",
        background: "#353535",
        color: "white",
        display: "flex",
        flexWrap: "wrap",
        justifyContent: "space-between",
        left: "0",
        position: "fixed",
        width: "100%",
        zIndex: "999"
      }
```
- in the `contentStyle` property pass:
```js
      contentStyle: {
        flex: "1 0 300px",
        margin: "15px"
      }
```

**Comment by Mastermindzh** [comment]
Hey Abby,

I'm happy to report that in commit [e8d4dca](https://github.com/Mastermindzh/react-cookie-consent/commit/0a5e4ea4408847aeaf2b2b52ccb349b21b0dccae) I've implemented this property.
It's available on NPM right now in [version 2.1.0](https://www.npmjs.com/package/react-cookie-consent/v/2.1.0)

Happy coding and don't forget to star the repo if you like the package :smile: 

cheers!
